src/Uno.UI/UI/Xaml/Application.iOS.cs

```diff
@@ -44,6 +44,11 @@ public Application(IntPtr handle) : base(handle)
 			callback(new ApplicationInitializationCallbackParams());
 		}
 
+		public void Exit()
+		{
+			NSThread.Exit();
```

Comment:
Using this will result in the app being rejected from the store.

Reply:
It may cause the rejection, but not certain 😂 . Microsoft Teams on iOS uses a form of this apparently https://stackoverflow.com/q/56443062/732221. Maybe it should be added to docs that this is in general discouraged? Even the UWP API is [not supposed to be used](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.application.exit?view=winrt-19041#remarks).

Reply:
I'm using this (exit) method in catch block - to write to something like error.log, and (if cannot recover from this state), exit/close app.